runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -5086,6 +5089,17 @@ static void jitStateLogic(J9JITConfig * jitConfig, TR::CompilationInfo * compInf
             }
          }
       }
+   else if (!alreadyIssuedLateDisclaim
+            && (crtElapsedTime - persistentInfo->getJitStateChangeTime() >= 120000)
```

Comment:
We can refine this later. Let's say that the application is under load and switches back and forth between rampup and stead-state - which I have seen happening. We may never trigger a disclaim operation.
Maybe it's better to compute the cumulative time spent in rampup/stead/deep-steady and don't care about the order of these phases or if they get interspersed with idle phases.